lib/index.js

```diff
@@ -66,6 +65,7 @@ PG.prototype.connect = function(config, callback) {
   config.idleTimeoutMillis = config.idleTimeoutMillis || config.poolIdleTimeout || defaults.poolIdleTimeout;
   config.log = config.log || config.poolLog || defaults.poolLog;
 
+  var poolName = JSON.stringify(config || {});
```

Comment:
I'd suggest doing it in two commits submitted as one PR (so you can just update this PR). The first would move the location of JSON.stringify call. The second would add json-stable-stringify and replace the stringify call with the stable one.